AzureStack/2015-04-01/swagger/GalleryAdmin.json

```diff
@@ -0,0 +1,502 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "GalleryAdminClient",
+    "description": "Management client for azurestack gallery admin operations.",
+    "version": "2015-04-01"
+  },
+  "host": "publicapi.azurestack.local",
```

Comment:
Nope, the APIs exposed by this spec do not go against public azure, they are relevant only for azurestack deployment